app/assets/stylesheets/app/mailers/layout.scss

```diff
@@ -4,6 +4,89 @@
 
 body {
   padding: 5px;
-  color: $m-mail-font-color;
+  color: $m-grey-email-color;
   line-height: 1.5em;
-}
+  background: $m-background;
+  font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  word-wrap: break-word;
+
+  .header-logo {
+    text-align: center;
+  }
+  .header_image {
+    text-align: center;
+    margin-bottom: 1.5em;
+    img {
+      width: 100%;
+      height: auto;
+    }
+  }
+  .header_message {
+    color: $m-mail-header-footer-color;
+    text-align: center;
+    font-size: 22px;
```

Comment:
Please use definitions for font-sizes (for example "$m-font-fields: 22px;"), you could create a definition specific for e-mail font-sizes.

Reply:
changed to: $m-mail-header-footer-font-size